cumulus/parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -889,7 +891,49 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// Migrations to apply on runtime upgrade.
-pub type Migrations = (pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,);
+pub type Migrations = (
+	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
+	pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
```

Comment:
Where did the Multisig migration come from?

Reply:
What do you mean where did it come from?

There was a change to the storage layout of the `Multisig` pallet and its version was updated from 0 to 1.

This migration was never applied to the `asset-hub-kusama` or `asset-hub-westend` runtimes.

Reply:
Right, sorry, was looking for it in this PR.